src/kafka/kafka.ts

```diff
@@ -0,0 +1,55 @@
+import { Consumer } from "kafkajs";
+import { DBOSContext } from "..";
+import { MethodRegistration, registerAndWrapFunction } from "../decorators";
+import { DBOSExecutor } from "../dbos-executor";
+import { Transaction } from "../transaction";
+import { Workflow } from "../workflow";
+
+export class KafkaRegistration<This, Args extends unknown[], Return> extends MethodRegistration<This, Args, Return> {
```

Comment:
Is there a way we can put the Kafka code in it's own package (under `/pacakges`) instead of integrating it into the core? I suspect there will be more of these integrations in the future

Reply:
This one is very lightweight (732 kB of dependencies total, <100 LoC), so I'm fine keeping it in the main package.  We'll see how it develops as we do more integrations--don't want a million sub-packages, don't want to bloat the main package.